src/components/VideoPlayer.js

```diff
@@ -56,8 +69,11 @@ class VideoPlayer extends React.Component {
     }
 
     return (
-      <div className="aspect-w-16 aspect-h-9" id="player">
-        <YouTube videoId={this.state.song} opts={opts} onReady={this._onReady} onStateChange={this.onYoutubeStateChange}/>
+      <div>
+        <div className="aspect-w-16 aspect-h-9" id="player">
+          <YouTube videoId={this.state.song} opts={opts} onReady={this._onReady} onStateChange={this.onYoutubeStateChange}/>
+        </div>
+        <button onClick={this.becomeAdmin}>Become Admin</button>
```

Comment:
So anyone can become admin and the server no longer handles this?

Reply:
Anyone can become admin because if the original admin leaves, no one is admin and the new song will never be played. Pressing the buttonw ill make the person pressing the button admin and make the current admin not admin. The server handles the romval of the old admin